src/Objs/Homepage/HomepageEditingConfig.ts

```diff
@@ -86,7 +91,7 @@ provideEditingConfig(Homepage, {
     },
     ...defaultPagePropertiesGroups,
   ],
-  properties: defaultPageProperties,
+  properties: [...defaultPageProperties, 'linkIcon'],
```

Comment:
* [ ] FYI we could think about putting linkIcon to defaultPageProperties to allow for product pages in navs, too.